USAGE.md

```diff
@@ -106,6 +106,7 @@ The `Watcher` constructor can be passed 3 different options:
 * `time` - The time threshold
 * `ratio` - The ratio threshold
 * `rootMargin` - The [rootMargin](https://wicg.github.io/IntersectionObserver/#dom-intersectionobserverinit-rootmargin) in object form.
+* `root` - Element with respect to which we want to watch the target. By default it is window.
```

Comment:
Might be helpful to link to the spec definition https://wicg.github.io/IntersectionObserver/#intersectionobserver-intersection-root